autoload/sy.vim

```diff
@@ -24,9 +31,10 @@ function! sy#start() abort
     return
   endif
 
-
   function! s:chdir()
```

Comment:
Yeah, this should all be handled by a `try/finally` around where we actually need to perform the diff.